### ISSUE TITLE
About code

### ISSUE THREAD

**Issue by whutWuXiaofeng** [sjruan/DeepMG#2]
Hello, excuse me, I am very interested in this paper of yours, but when I run the code, I found that the geometry_tanslation module calls the load_rn_shp function. Does this mean that the shp data set is required, but store_rn_shp is defined, but not Is called, can you help me with my doubts, thank you very much

**Comment by sjruan** [comment]
Yes, the shp data set is required. It is the data format used to store the road network. The OSM data can be converted to the shp format using [osm2rn](https://github.com/sjruan/osm2rn).
